File: C++ Programming/Swaping.cpp
#include<iostream>
#include<conio.h>
using namespace std;
int main()
{
    int a,b,temp;
    cout<<"Enter the Value of A "<<endl;
    cin>>a;
    cout<<"Enter the Value Of B "<<endl;
    cin>>b;

    cout<<"Before Swaping Values are A = "<<a<<" and B = "<<b<<endl;
    
    temp = a;
    a = b;
    b = temp;

    cout<<"After Swaping Values are A = "<<a<<" and B = "<<b<<endl;

    getch();
    return 0;
}

File: Java Programming/Swaping.java
import java.util.Scanner;
class Swaping 
{
        public static void main(String [] args)
        {
            Scanner sc = new Scanner(System.in);
            int a = sc.nextInt();
            int b = sc.nextInt();
            int temp;
            System.out.println("Before swaping a="+a);
            System.out.println("Before swaping b="+b);
            temp=a;
            a=b;
            b=temp;
            System.out.println("After swaping a="+a);
            System.out.println("After swaping b="+b);
            
        }    
}
